packages/yew-macro/src/props/prop.rs

```diff
@@ -54,6 +54,30 @@ impl Parse for Prop {
     }
 }
 
+fn strip_braces(expr: Expr) -> syn::Result<Expr> {
+    match expr {
+        Expr::Block(ExprBlock { block: Block { mut stmts, .. }, .. }) if stmts.len() == 1 => {
+            let stmt = stmts.remove(0);
+            match stmt {
+                Stmt::Expr(expr) => Ok(expr),
+                Stmt::Semi(_expr, semi) => Err(syn::Error::new_spanned(
+                        semi,
+                        "only an expression may be assigned as a property. Consider removing this semicolon",
+                )),
+                _ =>             Err(syn::Error::new_spanned(
+                        stmt,
+                        "only an expression may be assigned as a property",
+                ))
+            }
+        }
+        Expr::Lit(_) | Expr::Block(_) => Ok(expr),
+        _ => Err(syn::Error::new_spanned(
+                &expr,
+                format!("the property value must be either a literal or enclosed in braces. Consider adding braces around your expression like this: {{ {} }}", expr.to_token_stream()),
```

Comment:
```suggestion
                format!("the property value must be either a literal or enclosed in braces. Consider adding braces around your expression like this: {{{}}}", expr.to_token_stream()),
```
This might be a bit too pedantic, however, this follows the style of all the changes made in this PR.
Following this message might cause users to think that spaces in the braces is the 'correct' format while all the docs are using snippets without spaces.